add-on/panels/js/sidebar.jsx

```diff
@@ -156,6 +156,9 @@ loop.sidebar = (function(mozL10n) {
         dataDriver: dataDriver
       });
 
+      dispatcher.dispatch(
+        new sharedActions.SetOwnDisplayName({ displayName: "Wanda RoomOwner" }));
```

Comment:
It should use the username pref, I've got some code for that, I was thinking I'd put it in a spin-off bug.  I'll add XXXakita's with pointers to that bug.